### DISCUSSION TITLE
Get Specific Email From An Object ID

### DISCUSSION THREAD

**Discussion by azreal1231** [O365/python-o365#790] [Q&A]
So when you request emails from your inbox you get emails but and in each specific email there is an [object_id]
what i wanna do it save that [object_id] and use it to get the email its associated to at a later stage?
I am assuming I'd have to use the query but I am not sure how?

**Comment by alejcas** [comment]
I don’t remember if we have a “get email by object id” function. I would say we don’t have that and should be implemented but check the code yourself in case it exists

**Reply by azreal1231** [reply]
Hi, so you actually do had to dive into other docs to find it lol
code is

 **mailbox = account.mailbox()
 raw_data = mailbox.get_message(_object_id)**
 
 just adding this here for others
